src/data/execution-bounty-hunters.csv

```diff
@@ -0,0 +1,52 @@
+username, name, score
+samczsun, "Sam Sun", 35000
+holiman, "Martin Holst Swende", 33500
+chainsecurity, "ChainSecurity", 21000
+junorouse, "Juno Im", 20500
+uknowy, "Yoonho Kim (team Hithereum)", 20000
+johnyangk, "John Youngseok Yang (Software Platform Lab)", 20000
+guidovranken, "Guido Vranken", 18000
+peckshield, "PeckShield", 17000
+itsunixiknowthis, "ItsUnixIKnowThis", 15000
+catageek, "Bertrand Masius", 15000
+tintinweb, "Tin", 12500
+none, "Ralph Pichler", 12500
```

Comment:
This `none` value currently creates a broken image in our leaderboard:
![Image 2022-04-24 at 8 26 01 PM](https://user-images.githubusercontent.com/8097623/164990984-87dfaa5e-ad36-4863-8548-655667107f6b.jpg)

We should add a fallback to that Leaderboard component for the case they don't have a GH profile.